src/lib/specialistBios/data.json

```diff
@@ -0,0 +1,122 @@
+[
```

Comment:
`lib/specialistBios` feels like an odd place for this kind of content static content. Can we use the existing `src/data` folder for this? And if we need more fine-grained structure we could create a new folder in there.

Reply:
thanks, solved!